I have upgraded our build environment for https://github.com/datalad/git-annex from bullseye (now old stable). to bookworm (now stable). Docker images are at https://hub.docker.com/repository/docker/datalad/buildenv-git-annex/tags (prior ones at https://hub.docker.com/repository/docker/datalad/buildenv-git-annex-buster/tags) .

The git-annex (`10.20230828+git2-g50300a47fe-1~ndall+1`) has built fine and all testing passed, but one [extra test has failed](https://github.com/datalad/git-annex/actions/runs/6029623735/job/16366573859#step:6:33):

```
>> nfailed version 'libpcre.*so'
>> strace -f git-annex version
>> tee /dev/fd/2
>> wc -l
>> awk '/libpcre.*so.*ENOENT/{print}'
[pid  1843] openat(AT_FDCWD, "/usr/lib/git-annex.linux//lib/x86_64-linux-gnu/glibc-hwcaps/x86-64-v4/libpcre2-8.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
[pid  1843] openat(AT_FDCWD, "/usr/lib/git-annex.linux//lib/x86_64-linux-gnu/glibc-hwcaps/x86-64-v3/libpcre2-8.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
[pid  1843] openat(AT_FDCWD, "/usr/lib/git-annex.linux//lib/x86_64-linux-gnu/glibc-hwcaps/x86-64-v2/libpcre2-8.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
[pid  1843] openat(AT_FDCWD, "/usr/lib/git-annex.linux//lib/x86_64-linux-gnu/tls/haswell/libpcre2-8.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
[pid  1843] openat(AT_FDCWD, "/usr/lib/git-annex.linux//lib/x86_64-linux-gnu/tls/libpcre2-8.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
[pid  1843] openat(AT_FDCWD, "/usr/lib/git-annex.linux//lib/x86_64-linux-gnu/haswell/libpcre2-8.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
> test 6 -lt 5
Error: Process completed with exit code 1.
```

[previously (`10.20230828-1~ndall+1`was the last successful build) it was just 3](https://github.com/datalad/git-annex/actions/runs/6003338461/job/16282128434):

```
>> nfailed version 'libpcre.*so'
>> strace -f git-annex version
>> tee /dev/fd/2
>> wc -l
>> awk '/libpcre.*so.*ENOENT/{print}'
[pid  1932] openat(AT_FDCWD, "/usr/lib/git-annex.linux//lib/x86_64-linux-gnu/tls/haswell/libpcre2-8.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
[pid  1932] openat(AT_FDCWD, "/usr/lib/git-annex.linux//lib/x86_64-linux-gnu/tls/libpcre2-8.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
[pid  1932] openat(AT_FDCWD, "/usr/lib/git-annex.linux//lib/x86_64-linux-gnu/haswell/libpcre2-8.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
> test 3 -lt 5
```

where those 3 extra (identical) looks up are coming from now?

PS Since not critical, I will workaround for now by raising max count to 7.
